### DISCUSSION TITLE
Avalonia  and VS 2022 Comunity

### DISCUSSION THREAD

**Discussion by moiseyo** [AvaloniaUI/Avalonia#15147] [Q&A]
I'm using Vs 2022 community
After instruction Avalonia used AvaloniaVS.VS2022.vsix, restating VS, and rebooting the computer new project does not have  Avalonia.
Using a sample   project gives me the error   no found Avalonia  
I can see  Avalonia  in the extension and  Nuget 
My Computer is W10 
What I'm missing

**Comment by wsficke** [comment]
Did you check it is enabled in Manage Extensions window in VS?

**Reply by gabrielopeculiar** [reply]
I´m just get a backlink. [resenha de livros](https://www.mentesemcorrente.com.br/)

**Comment by moiseyo** [comment]
Yes 
![Untitled](https://github.com/AvaloniaUI/Avalonia/assets/5314730/9636ca89-5a60-41f6-b090-fd035582b4bf)
I attached  screen short

**Comment by moiseyo** [comment]
![Untitled](https://github.com/AvaloniaUI/Avalonia/assets/5314730/85245178-15de-46d1-9c7a-c3dc18f1cb9e)

New File Scrreen

**Comment by jp2masa** [comment]
Did you try installing the templates using the dotnet CLI?

```
dotnet new install Avalonia.Templates
```

**Comment by moiseyo** [comment]
![image](https://github.com/AvaloniaUI/Avalonia/assets/5314730/1462ea76-df58-40e5-9ba5-b6b9971a66d8)

I tried
![Untitled](https://github.com/AvaloniaUI/Avalonia/assets/5314730/1ca6b262-9e83-403b-91e5-20c20ee9561b)
    no effect

**Comment by workgroupengineering** [comment]
Uninstall Avalonia Extensions and install [this pre-release](https://dev.azure.com/AvaloniaUI/aa84306f-2981-47b9-8206-edb3bed6250d/_apis/build/builds/46670/artifacts?artifactName=publish&api-version=7.1&%24format=zip).

**Reply by moiseyo** [reply]
still  not runnig I ran  command  dotnet new install Avalonia.Templates

**Reply by workgroupengineering** [reply]
Plese try with following steps
 - Uninstall Avalonia Extensions
 - Find  thne `AvaloniaVS.VS2022.pkgdef` file in folder `%LocalAppData%\Microsoft\VisualStudio\` . You should find a path similar to: `C:\Users\YourUser\AppData\Local\Microsoft\VisualStudio\17.0_5dfd1996\Extensions\blrvrhr5.n4k\AvaloniaVS.VS2022.pkgdef`
 - Remove the folder you found. eg: `C:\Users\YourUser\AppData\Local\Microsoft\VisualStudio\17.0_5dfd1996\Extensions\blrvrhr5.n4k`
 - run PowerShell and type `dotnet new uninstall Avalonia.Templates` and press the Enter key
 - install [this pre-release](https://dev.azure.com/AvaloniaUI/aa84306f-2981-47b9-8206-edb3bed6250d/_apis/build/builds/46670/artifacts?artifactName=publish&api-version=7.1&%24format=zip).

**Comment by thevortexcloud** [comment]
Does it generate a project if you do `dotnet new avalonia.app` in a terminal?

**Comment by moiseyo** [comment]
I done
result
[image: image.png]
Moisey O

**Comment by moiseyo** [comment]
[image: image.png]
Moisey O

**Comment by moiseyo** [comment]
Is the problem with the Comunity edition Visual Studio or Win10 Prof
Moisey O

**Comment by moiseyo** [comment]
I completely reinstall  VS 2022  Comunity  on  w10 Prof
and no nay success
Moisey O
